src/api/app/assets/javascripts/webui/add_file.js

```diff
@@ -0,0 +1,54 @@
+function createInputs(item) {
+  var filelist = $(item.list);
+  var li = document.createElement('li');
+  li.setAttribute('class', 'd-flex list-group-item align-items-center');
+  var i = document.createElement('i');
+  i.setAttribute('class', 'fa fa-fw mr-3 fa-' + item.icon);
+  li.appendChild(i);
+  item.inputs.forEach(function(input) {
+    var field = document.createElement('input');
+    field.value = input.value;
+    field.name = input.name;
+    field.setAttribute('type', input.type);
+    field.setAttribute('class', 'form-control mr-3');
+    field.setAttribute('required', 'true');
+    field.setAttribute('placeholder', input.placeholder);
+    li.appendChild(field);
+  });
+  if(item.remove) {
+    var remove = document.createElement('i');
+    remove.setAttribute('class', 'fa fa-times text-danger');
+    remove.id = 'remove-row';
+    li.appendChild(remove);
+  }
+  filelist.append(li);
+}
+
+function toggleUpload(value) {
```

Comment:
:thought_balloon: Is this `value` the number of files? Can we find a better name?